### ISSUE TITLE
I almost got the installation. But met with an error at the end I guess. I got an error in the terminal. Please find the attached screenshot below.

### ISSUE THREAD

**Issue by KND6451** [ajay6763/LitMod2D_2.0_package_dist_users#3]
![Screenshot from 2019-10-15 18-10-01](https://user-images.githubusercontent.com/52480430/66834471-49d19a80-ef7b-11e9-9a24-7da27c340534.png)

**Comment by prakashncess** [comment]
I tried sudo apt-get install libcanberra-gtk-module but still error is there
![Uploading Screenshot from 2021-09-24 02-45-47.png…]()

**Comment by ajay6763** [comment]
I see that these are just warnings. May be try to proceed buliding a model and see if it is working fine.

**Comment by prakashncess** [comment]
I tried buidling model with this warning but python crashed error popup ..see screenshot
![Python error](https://user-images.githubusercontent.com/36472232/134465350-4b275c65-bcf3-4be0-a446-ea0f17ebcfb5.png)

**Comment by prakashncess** [comment]
Now I am getting error like segmentation core dumped..plz see screenshot
![Segmentation Core dumped](https://user-images.githubusercontent.com/36472232/134465998-088b5b96-83ae-45ea-9551-848a8e220d99.png)

**Comment by ajay6763** [comment]
I can see that it is giving errors about the backend. I

You can try a small hack below.
Go to the following file and change self.toolkit to qt4
/usr/lib/python/dist-packages/traits/etsconfig/etsconfig.py

in this file change the

change self._toolkit = "qt4"

let me know if it works

**Comment by prakashncess** [comment]
I found many places self._toolkit which i removed with qt4. I get this error now
![qt4 error](https://user-images.githubusercontent.com/36472232/134468487-3a2bccfc-45ba-4123-a6b6-943703945d40.png)

**Comment by ajay6763** [comment]
No. There will be self._toolkit = " " . Indside the double quotes put qt4 .

**Comment by prakashncess** [comment]
I replaced it in place near to backend....now segmentation fault (core dumped) error is showing while building model.
![Segmentation error](https://user-images.githubusercontent.com/36472232/134469818-bf0b0788-9b18-492b-8ec3-e5960d8dd9c6.png)

**Comment by ajay6763** [comment]
@prakashncess  I see, I will have to look into this error in detail. Will get back to you asap. Could you please attach the list of libraries and their version that you installed? 
 
pip list > liberaries.txt

**Comment by prakashncess** [comment]
I have solved the issue and program is successfully installed. I just changed activation style to work in same environment:
I changed [source ./LitMod2D 2.0/bin activate] to [source ./LitMod2D 2.0/bin/activate] and it worked. 

But I do not see any LitMod_test_ model in my running directory to test the installation.

**Comment by ajay6763** [comment]
Perfect! You can find it here. https://drive.google.com/file/d/1WIyLfrHR_TFJhH_JpEefOI1VZFBub3uG/view?usp=drive_web

**Comment by ajay6763** [comment]
@prakashncess  could you please attach the list of libraries from your environment here and your OS name here so that future users can get help from this. Also if it is possible to list the steps that you have taken to install the pyside.

Thanks for your patience!

**Comment by prakashncess** [comment]
Sure, after checking proper working of Litmod, I will certainly put here. I am unable to download the test model from the given link.

**Comment by prakashncess** [comment]
Thanks Dr. Ajay for test model and I checked, it is working perfectly. Thank you so much for your consistent support and guidance. 
I hope my own problem of profile modeling will go smoothly.

Cheers!
